Helper/Config.php

```diff
@@ -288,6 +288,10 @@ class Config extends AbstractHelper
         self::SUCCESS_PAGE_ACTION
     ];
 
+    public static $supportableProductTypes = [
```

Comment:
This might be a little vague. What if we introduce another checkout feature and need separate supportable product type configs?

Reply:
Renamed to $supportableProductTypesForProductPageCheckout